### PR TITLE
Logging refactor

### PR DESCRIPTION
**Pull request by kdiluca**
No longer storing the start time since we're now logging at each level of the service.

### CODE REVIEW

**Comment by kevinkreiser** [comment]
Ship. But curios if you can just subtract start time from end time and save the duration cast

File: src/loki/service.cc [hunk 1]
```diff
@@ -183,6 +186,19 @@ namespace valhalla {
         http_response_t response(501, "Not Implemented", "Not Implemented", headers_t{CORS});
         response.from_info(info);
         result.messages.emplace_back(response.to_string());
+
+        //get processing time for loki
+        auto end_time = std::chrono::high_resolution_clock::now();
+        auto elapsed_time = static_cast<float>(std::chrono::duration_cast<std::chrono::milliseconds>(end_time.time_since_epoch() - start_time.time_since_epoch()).count());
```
Comment (kdiluca) [inline src/loki/service.cc]: Wasn't able to remove the duration_cast logic so I just combined everything instead.

**Comment by kevinkreiser** [comment]
can we switch to using `system_clock` instead? we are only working in milliseconds so we shouldn't need `high_resolution_clock` and the extra overhead may bring

**Comment by kevinkreiser** [comment]
instead of epoch seconds you can get a duration just by using subtraction, and you can tell it to be float:


    auto s = std::chrono::system_clock::now();
    auto e = std::chrono::system_clock::now();
    std::chrono::duration<float, std::milli> d = e - s;
    std::cout << d.count() << std::endl;

here `d.count()` is `float`

**Comment by kdiluca** [comment]
Sure, I'll do that instead.

**Comment by kevinkreiser** [comment]
remove cout then :rocket:

**Pull request merged**